### PR TITLE
Coveralls in search page

### PR DESCRIPTION
**Pull request by mihaidma**
Coveralls integrated in search page

### CODE REVIEW

**Comment by mihaidma** [comment]
@ckiss could you please review?

File: client/components/module-source.js [hunk 1]
```diff
@@ -1,21 +1,21 @@
 'use strict'
 
 import React from 'react'
+import ClassName from 'classnames'
```
Comment (ckiss) [inline client/components/module-source.js]: I use to name this classNames :) I don't see the need to use pascal case

File: client/components/module-source.js [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
     data: React.PropTypes.object.isRequired,
-    sourceName: React.PropTypes.string.isRequired
+    sourceName: React.PropTypes.string.isRequired,
+    imgSrc: React.PropTypes.string
   },
   render () {
-    let className = `logo logo-${this.props.sourceName} ${this.props.className}`
-    let source = this.props.data[this.props.sourceName]
-    let url = source ? source.url : ''
+    let {imgSrc, sourceName, className, data: {[sourceName]: source}, data: {[sourceName]: {url: url}}} = this.props
```
Comment (ckiss) [inline client/components/module-source.js]: this is real good stuff, but might be confusing after all (for someone who sees this for the first time :D )

File: client/components/module-source.js [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+    let logoClass = ClassName({[`logo logo-${sourceName} ${className}`]: !imgSrc})
+    let img = imgSrc && <img src={imgSrc} alt={sourceName} className="img-badge"/>
```
Comment (ckiss) [inline client/components/module-source.js]: I would put this two guys(logoClass and img) inline. they are not used anywhere else, so I don't see the need of declaring new variables :)

**Comment by ckiss** [comment]
@mihaidma few very small things, otherwise LGTM. feel free to merge if you don't agree with my remarks 👍

File: client/components/module-source.js (hunk 1 above)
Comment (mihaidma) [inline client/components/module-source.js]: :+1:

**Pull request merged**